pretrained_models/models.py

```diff
@@ -20,10 +21,16 @@ def get_model(model_name):
         return keras.applications.ResNet50V2(weights="imagenet", include_top=False)
     if model_name == "inceptionV3":
         return keras.applications.InceptionV3(weights="imagenet", include_top=False)
-    if model_name == "googlenet":
+    if model_name == "inceptionV1":
         tf.compat.v1.keras.backend.set_image_data_format('channels_first')
         return googlenet.create_googlenet()
     if model_name == "inceptionV1slim":
         inputs = tf.random.uniform((1, 224, 224, 3), dtype=tf.dtypes.float32)
         return slim_nets.inception.inception_v1(inputs)
-    return keras.applications.VGG16(weights="imagenet", include_top=False)
+    if model_name == "VGG16":
+        return keras.applications.vgg16(weights= "imagenet", include_top=False)
+    if model_name == "alexnet":
+        return alexnet.AlexNet()
+    else:
+        print("Model is not specified")
```

Comment:
The problem about this is, that we do have one path now that does not return anything. I put a default return here that returns VGG16 if nothing else matches. Maybe we should go back to that and print a warning? Otherwise the code will fail if the wrong model is specified. In the long run, we might want to have individual functions for the models to get rid of this `if/else`, which is not very pretty.

Reply:
We can talk about it. I just don't like the VGG as fallback but we can decide about this.

Reply:
Sure, lets talk about this tomorrow.